Procfile.example

```diff
@@ -0,0 +1,48 @@
+# Web server workers
+# The $PORT variable is set automatically by foreman for particular workers
+# We tell foreman to start at 3000, it then increments by 1 per instance of a worker type
+# and 100 per worker type.
+#
+# So, the first ui worker will get PORT=3000 while the next will get PORT=3001
+# The first api worker will get PORT=3100 and the next will get PORT=3101
+
+# ui:        env PORT=$PORT bundle exec rails r lib/workers/bin/run_single_worker.rb MiqUiWorker
+# api:       env PORT=$PORT bundle exec rails r lib/workers/bin/run_single_worker.rb MiqWebServiceWorker
+# websocket: env PORT=$PORT bundle exec rails r lib/workers/bin/run_single_worker.rb MiqWebsocketWorker
```

Comment:
Confirmed.  Had to change the `lib/workers/bin/run_single_worker.rb` to set the `ENV["BUNDLER_GROUPS"` with a `||=`, but after that, it worked just fine using vanilla `ruby script arg`.

Probably want to make a few changes to make sure that we can:

A) Make that `||=` a thing in the script so that the bundler groups are user modifyable
B) And/or make sure that the bundler groups in `lib/workers/miq_worker_types.rb` is correct in all cases.  Seems like the UiWorker needs all the things, aka `:manageiq_default`.